### PR TITLE
fix: handle private hook implementations

### PR DESCRIPTION
**Pull request by dmke**
This fixes a regression introduced in #402.

Details: https://github.com/gonzalo-bulnes/simple_token_authentication/pull/402#issuecomment-1369269890

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+
+### Fixed
+
+- Fix a regression introduced by #402 - @dmke
```
Comment (gonzalo-bulnes) [inline CHANGELOG.md:10]:
```suggestion
- Fix a regression introduced by #402, that caused hooks not to be invoked - @dmke
```

**Review by gonzalo-bulnes — APPROVED** [review]
This a very clean patch @dmke. If you could add a little more detail to the changelog entry, that would be great. Otherwise, on visual review, this looks ready to go and I see no reason to hold it.

File: CHANGELOG.md (hunk 1 above)
Comment (dmke) [reply CHANGELOG.md]: I have applied the detail.

**Review by gonzalo-bulnes — APPROVED** [review]

**Comment by gonzalo-bulnes** [comment]
Your fix is included in release **v1.18.1**, thank you @dmke ! :raised_hands:

**Comment by dmke** [comment]
Yay :tada:

Thank you very much!

**Pull request merged**